src/pages/alpha/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,75 @@
+import { Complete, Todo, Form } from './components';
+import { useEffect, useState } from 'react';
+
+const TodoList = () => {
+  const localList = JSON.parse(localStorage.getItem('alpha_list')) || [];
+  const [list, setList] = useState(localList);
+
+  function createItem(value) {
+    const item = { id: Date.now(), title: value, status: 'todo', state: 'null' };
+    console.log(list);
+    setList((list) => list.concat(item));
+    localStorage.setItem('alpha_list', JSON.stringify(list));
+    // event.target.reset();
+  }
+
+  function onSubmit(event) {
+    event.preventDefault();
+    const formdata = new FormData(event.target);
+    createItem(formdata.get('todo'));
+  }
+
+  function toggleStatus(item) {
+    item['status'] === 'complete' ? (item['status'] = 'todo') : (item['status'] = 'complete');
+    console.log(item);
```

Comment:
這邊用 ternary operator 閱讀性會很差
要也是應該這樣寫
```suggestion
item.status = item.status === 'complete' ? 'todo' : 'complete';
```
所以說為什麼要用引號來取 properties?
不需要的 log 也不要存